src/Microsoft.SqlTools.ServiceLayer/Agent/AgentService.cs

```diff
@@ -1211,6 +1232,461 @@ public static XmlDocument CreateJobXmlDocument(string svrName, string jobName)
+                            {
+                                AppendLogToTable = false,
+                                AppendToLogFile = false,
+                                AppendToStepHist = false,
+                                Command = execNotebookScript,
+                                CommandExecutionSuccessCode = 0,
+                                DatabaseName = "",
+                                DatabaseUserName = null,
+                                FailStepId = 0,
+                                FailureAction = StepCompletionAction.QuitWithFailure,
+                                Id = 1,
+                                JobId = null,
+                                JobName = parameters.Notebook.Name,
+                                OutputFileName = null,
+                                ProxyName = null,
+                                RetryAttempts = 0,
+                                RetryInterval = 0,
+                                Script = execNotebookScript,
+                                ScriptName = null,
+                                Server = "",
+                                StepName = "Exec-Notebook",
+                                SubSystem = AgentSubSystem.PowerShell,
+                                SuccessAction = StepCompletionAction.QuitWithSuccess,
+                                SuccessStepId = 0,
+                                WriteLogToTable = false,
+                            };
+                            parameters.Notebook.JobSteps = new AgentJobStepInfo[1];
+                            parameters.Notebook.JobSteps[0] = notebookJobStep;
+                            var result2 = await ConfigureAgentJob(
+                                parameters.OwnerUri,
+                                parameters.Notebook.Name,
+                                parameters.Notebook,
+                                ConfigAction.Create,
+                                ManagementUtils.asRunType(parameters.TaskExecutionMode));
+                            ConnectionInfo connInfo;
+                            ConnectionServiceInstance.TryFindConnection(parameters.OwnerUri, out connInfo);
+                            string templateFilePath = parameters.TemplateFilePath;
+                            string templateFileContents = File.ReadAllText(templateFilePath);
+                            string targetDatabase = parameters.Notebook.TargetDatabase;
+                            string notebookDatabaseSetupQueryString =
+                            @"
+                            IF NOT EXISTS (
+                            SELECT  schema_name
+                            FROM    information_schema.schemata
+                            WHERE   schema_name = 'notebooks' ) 
+                            BEGIN
+                            EXEC sp_executesql N'CREATE SCHEMA notebooks'
+                            END
+
+                            IF  NOT EXISTS (SELECT * FROM sys.objects 
```

Comment:
`IF  NOT EXISTS (SELECT * ` could be `IF  NOT EXISTS (SELECT TOP 1 id` or something